src/test/common/CMakeLists.txt

```diff
@@ -152,6 +152,12 @@ add_executable(unittest_tableformatter
 add_ceph_unittest(unittest_tableformatter ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/unittest_tableformatter)
 target_link_libraries(unittest_tableformatter global)
 
+add_executable(unittest_xmlformatter
+    test_xmlformatter.cc
+    )
+add_ceph_unittest(unittest_xmlformatter ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/unittest_xmlformatter)
+target_link_libraries(unittest_xmlformatter global)
```

Comment:
hi, @tchaikov , what's `global` and diff with `ceph-common`, i saw that all unittest use `global` in this CMakeLists.txt. Need we update other test cases?